pkg/scheduler/scheduling_queue_test.go

```diff
@@ -827,3 +1118,157 @@ func TestIsEmpty(t *testing.T) {
 	leaf.addSchedulingApplication(app)
 	assert.Equal(t, leaf.isEmpty(), false, "queue with registered app should not be empty")
 }
+
+func TestGetOutstandingRequest(t *testing.T) {
+	const app1ID = "app1"
+	const app2ID = "app1"
```

Comment:
The logic for calculating this is in the app and not in the queue.
If the app returns the correct thing the queue just add them at this point. We should test the app not the queue.

Reply:
we can add more tests for `app.getOutstandingRequests()`, but this is covering more code paths by including the queue, I suggest to keep it.

Reply:
we'll take this up via the follow up jira